nvflare/job_config/script_executor.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
This file does not belong to nvflare.job_config. It should go to somewhere in nvflare.app_opt.